### DISCUSSION TITLE
Soft Fail Attempt to Send Repeatedly

### DISCUSSION THREAD

**Discussion by TR-SametOzen** [postalserver/postal#3140] [Help with using Postal] [answered]
Hello,

If an email is marked as Soft Fail, I want it to become Hard Fail after 5 attempts, but it's been the 18th time and it's still trying to send. How can I fix this?

<img width="1170" alt="image" src="https://github.com/user-attachments/assets/56af3454-f16c-43b3-8bfb-7dda12c2cbaf">

**Comment by willpower232** [comment, marked as answer]
If it tried a 19th time, I'd be worried but in theory this is the final attempt.

I believe that number can be reduced with https://github.com/postalserver/postal/blob/da90e75036c27482699921613d838f4058a100e7/doc/config/yaml.yml#L13 (or https://github.com/postalserver/postal/blob/da90e75036c27482699921613d838f4058a100e7/doc/config/environment-variables.md?plain=1#L11) depending on your config situation.

**Reply by TR-SametOzen** [reply]
Hello,

I added it as follows. I hope it works. 

![image](https://github.com/user-attachments/assets/af2462a7-bb29-4bf7-ac34-77462017605b)

"postal/doc/config/environment-variables.md"  I couldn't find this file.

**Reply by willpower232** [reply]
as long as you restart Postal to apply the change it should do, you are evidently using a config file, not environment variables, so no need to worry about it.

**Reply by TR-SametOzen** [reply]
Ahh Thank You, Superman :)

it is work!

![image](https://github.com/user-attachments/assets/c9d758fe-1fac-492a-b594-837fd6af51d7)
